examples/suit_update/README.md

```diff
@@ -0,0 +1,551 @@
+- [Setup][setup]
+  - [Signing key management][key-management]
+  - [Setup a wired device using ethos][setup-wired]
+    - [Provision the device][setup-wired-provision]
+    - [Configure the network][setup-wired-network]
+  - [Alternative: Setup a wireless device behind a border router][setup-wireless]
+    - [Provision the wireless device][setup-wireless-provision]
+    - [Configure the wireless network][setup-wireless-network]
+  - [Start aiocoap fileserver][start-aiocoap-fileserver]
+- [Perform an update][update]
+  - [Build and publish the firmware update][update-build-publish]
+  - [Notify an update to the device][update-notify]
+- [Detailed explanation][detailed-explanation]
+- [Automatic test][test]
+
+## Prerequisites
+[prerequisites]: #Prerequisites
+
+- Install python dependencies (only Python3.6 and later is supported):
+
+      $ pip3 install --user ed25519 pyasn1 cbor
+
+- Install aiocoap from the source
+
+      $ pip3 install --user --upgrade "git+https://github.com/chrysn/aiocoap#egg=aiocoap[all]"
+
+  See the [aiocoap installation instructions](https://aiocoap.readthedocs.io/en/latest/installation.html)
+  for more details.
+
+- add `~/.local/bin` to PATH
+
+  The aiocoap tools are installed to `~/.local/bin`. Either add
+  "export `PATH=$PATH:~/.local/bin"` to your `~/.profile` and re-login, or execute
+  that command *in every shell you use for this tutorial*.
+
+- Clone this repository:
+
+      $ git clone https://github.com/RIOT-OS/RIOT
+      $ cd RIOT
+
+- In all setup below, `ethos` (EThernet Over Serial) is used to provide an IP
+  link between the host computer and a board.
+
+  Just build `ethos` and `uhcpd` with the following commands:
+
+      $ make -C dist/tools/ethos clean all
+      $ make -C dist/tools/uhcpd clean all
+
+  It is possible to interact with the device over it's serial terminal as usual
+  using `make term`, but that requires an already set up tap interface.
+  See [update] for more information.
+
+## Setup
+[setup]: #Setup
+
+### Key Management
+[key-management]: #Key-management
+
+SUIT keys consist of a private and a public key file, stored in `$(SUIT_KEY_DIR)`.
+Similar to how ssh names its keyfiles, the public key filename equals the
+private key file, but has an extra `.pub` appended.
+
+`SUIT_KEY_DIR` defaults to the `keys/` folder at the top of a RIOT checkout.
+
+If the chosen key doesn't exist, it will be generated automatically.
+That step can be done manually using the `suit/genkey` target.
+
+### Setup a wired device using ethos
+[setup-wired]: #Setup-a-wired-device-using-ethos
+
+#### Configure the network
+[setup-wired-network]: #Configure-the-network
+
+In one terminal, start:
+
+    $ sudo dist/tools/ethos/setup_network.sh riot0 2001:db8::/64
+
+This will create a tap interface called `riot0`, owned by the user. It will
+also run an instance of uhcpcd, which starts serving the prefix
+`2001:db8::/64`. Keep the shell open as long as you need the network.
+
```

Comment:
Please re-add the command that configures the riot0 interface on the host:
```
sudo ip address add 2001:db8::1/128 dev riot0
```

It will be ok like this (I just tested it). It would be even better if you could update the sentence above 'Keep the shell open as long as you need the network' in 'Keep the shell open as long as the device as a terminal opened'
 (or something better of course).

Reply:
And sorry for the round-trips.

Reply:
I've re-added the address add comment, and added a note to the other section:

```Make sure to exit the "make term" instance from the next section *before*                                                          
exiting this, as otherwise the "riot0" device doesn't get cleaned up properly.```

Does it make sense as is?

Reply:
Maybe just replace "device" with "interface". Otherwise it's ok. You can squash directly.